src/main/java/com/nike/cerberus/cli/EnvironmentConfigToArgsMapper.java

```diff
@@ -185,30 +184,10 @@ private EnvironmentConfigToArgsMapper() {
     }
 
     private static List<String> getUploadCertFilesCommandArgs(EnvironmentConfig environmentConfig, String[] passedArgs) {
```

Comment:
It looks like passedArgs is not used here, should it be?

I see we were checking for --overwrite flag in the passedArgs. Are we still safeguarding against overwrite?

Reply:
You don't override certs anymore you can upload as many as you want and the latest will be used

Reply:
--overwrite has been removed from the command, so I can remove passed args here from the signature